examples/bch-example/src/app/bch.tx.example.ts

```diff
@@ -1,9 +1,9 @@
 import { TatumBchSDK } from '@tatumio/bch'
-import { REPLACE_ME_WITH_TATUM_API_KEY } from '@tatumio/shared-testing-common'
-
-const bchSDK = TatumBchSDK({ apiKey: REPLACE_ME_WITH_TATUM_API_KEY })
 
 export async function bchTransactionsExample() {
+  const bchSDK = TatumBchSDK({ apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab' })
+
+  // Prepare signed transaction
   const txData = await bchSDK.transaction.prepareSignedTransaction(
```

Comment:
examples should be in terms of send*, not prepare. Prepare are our own internal helpers, publically there should be send method used

Reply:
done